_sass/minima.scss

```diff
@@ -23,6 +23,9 @@ $table-text-align: left !default;
 // Width of the content area
 $content-width:    800px !default;
 
+$on-medium:        600px !default;
+$on-large:         800px !default;
```

Comment:
instead of adding these two declarations here, simply do the following after existing variables:
```scss
$on-medium:    $on-palm !default;
$on-large:     $on-laptop !default;
```